Makefile

```diff
@@ -26,8 +26,17 @@ deploy:
 demo-local:
 	cargo run --bin local
 
+GREEN := \033[32m
+RESET := \033[0m
```

Comment:
Should this be defined somewhere else? This could be passed by the user if it's defined here